### PR TITLE
Create service design at dxw

### PR DESCRIPTION
**Pull request by Marianne-dxw**
Adding a guide on how we approach service design at dxw.

### CODE REVIEW

File: guides/service design at dxw [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Service Design approach
```
Comment (erbridge) [inline guides/service design at dxw:1]: This file needs to be renamed to something like `service-deign-at-dxw.md`. The file extension is important.

File: guides/service design at dxw [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Service designers consider factors that contribute to the service’s **future success**. We can create guides to support its implementation, prepare training that staff may need in order to deliver the service, and recommend organisational structures to ensure it will be properly managed in the future.
+- We also consider factors that could limit the success of the service and help teams mitigate them through **risk management**.
+- Services never exist in a vacuum, and their success depends on good alignment with the technology and processes that surround them. Service designers help manage these **technical and non-technical interfaces**.
+
+## 1. **Beta, live and beyond…**
+- Service designers like to be involved throughout the beta and live phases of a new service to help ensure that **performance is tracked and outcomes measured**.  We help service teams set up Key Performance Indicators and the data streams required to track them.
+- In a similar way we can also **support teams running existing services** to identify and implement improvements.
```
Comment (cocadxw) [inline guides/service design at dxw:42]: + add this one - When those services require the transformation of an existing service the service designer plays an important role helping to define channel shift strategies for transitioning to a digital service.

File: guides/service design at dxw [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Service Design approach
+
+This is an outline of how we approach Service Design at dxw. Service Design can cross over a number of disciplines, so we've written this based on our experience of applying service design to projects at dxw.
+
+## 1 **Clarify the problem and set the direction**
+- As part of the project kickoff, service designers interrogate the brief set by the client. We facilitate a **problem framing** session to identify the intended outcomes of the work (though not their expected solutions), alongside the recognised needs and problems that have initiated the project. This session happens early in the inception phase so that the outcomes can inform the project roadmap.
```
[the remark this answers is not in the document]
Comment (Marianne-dxw) [reply guides/service design at dxw]: Clarified that problem framing will happen after inception questions. Also added link to problem framing template.

File: guides/service design at dxw [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## 1. **Imagine new service experiences** 
+- Service designers help the team transition **from research insights to service concepts**. We plan and facilitate workshops and other design activities. There are a number of methods that we may use at this stage, but all of them link concepts directly to value creation for both the user and the organisation.
+- **Co-design** is the gold-standard method to use during this ideation phase. We bring together stakeholders, subject matter experts, designers and where possible end users  to generate and discuss design ideas.  
+- As part of ideation sessions, service designers can facilitate the **prioritisation of concepts** to prototype the best concepts (or blends of concepts). We use tools such as heuristic reviews or the theory of change to reflect on whether the proposed solution meets the needs and initial intentions of the service.
```
[the remark this answers is not in the document]
Comment (Marianne-dxw) [reply guides/service design at dxw]: I'd say co-design is a more commonly used (and preferred?) term?

[the remark this answers is not in the document]
Comment (Marianne-dxw) [reply guides/service design at dxw]: Changed ideation phase to "exploratory phase"

File: guides/service design at dxw [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- We help multi-disciplinary teams to reach the best outcomes by ensuring insights and ideas are shared effectively and that different **workstreams are aligned**. We facilitate **collaborative decision-making** and help teams and wider stakeholders reach **consensus** on the way forward.
+
+## 1. **Prototype the service**
+- Before any prototyping begins, service designers may test the basis of the value proposition to ensure that the service works in theory. This check ensures it offers value to the end users/customers as well as the business.
```
[the remark this answers is not in the document]
Comment (Marianne-dxw) [reply guides/service design at dxw]: done

File: guides/service design at dxw (hunk 2 above)
Comment (Marianne-dxw) [reply guides/service design at dxw]: done

**Pull request merged**